config/all-in-one/values.yaml

```diff
@@ -62,15 +62,25 @@ cp4waiops:
   # The CP4WAIOps deployment profile, e.g.: x-small, small, large.
   # NOTE: The profile x-small is only for small PoC or demo. If you are looking
   # for official installation, use profile such as small or large instead.
-  profile: x-small
+  profile: small
 
   # AIManager specific configuration
   aiManager:
+    # Specify whether or not to install AIManager.
+    enabled: true
     # The instance name of CP4WAIOps.
     instanceName: aiops-installation
     # The namespace of CP4WAIOps.
     namespace: cp4waiops
 
+  eventManager:
+    ## Enable Event Manager
+    enabled: true
+    ## Use the fully qualified domain name (FQDN) to formulate the clusterDomain property, using the following formula:
+    clusterDomain: apps.clustername.*.*.com
```

Comment:
Is `apps.clustername.*.*.com` a default value which can be turned into hidden helm param, or if it's a placeholder that needs to be modified when kick off the install?

Reply:
It's a placeholder that `must` be modified by customer when kick off the install. It is the domain name of OCP cluster like `apps.test.cp.fyre.ibm.com`